src/ui-kit/Button.jsx

```diff
@@ -1,13 +1,16 @@
 import { Button, makeStyles } from "@material-ui/core";
 
-const useStyles = makeStyles({
+const useStyles = makeStyles((theme) => ({
   flexGrow: {
     flex: "1",
   },
   button: {
     backgroundColor: "var(--light-gray)",
     color: "#000000",
     width: "327px",
+    [theme.breakpoints.down("xs")]: {
+      width: "240px",
```

Comment:
Do you think we should change this to viewWidth `vw`or a percentage instead of hard coded pixels? Just a thought, but I'm fine either way